SimTKcommon/Geometry/src/DecorativeGeometry.cpp

```diff
@@ -371,6 +371,104 @@ const std::string& DecorativeMeshFile::getMeshFile() const {
 }
 
 
+/////////////////////
+// DECORATIVE TORUS //
+/////////////////////
+
+DecorativeTorus::DecorativeTorus(Real torusR, Real tubeR) {
+    rep = new DecorativeTorusRep(torusR, tubeR);
+    rep->setMyHandle(*this);
+}
+
+Real DecorativeTorus::getTorusRadius() const {
+    return DecorativeTorusRep::downcast(*rep).getTorusRadius();
+}
+
+Real DecorativeTorus::getTubeRadius() const {
+    return DecorativeTorusRep::downcast(*rep).getTubeRadius();
+}
+
+DecorativeTorus& DecorativeTorus::setTorusRadius(Real torR) {
+    DecorativeTorusRep::downcast(*rep).setTorusRadius(torR); return *this;
+}
+DecorativeTorus& DecorativeTorus::setTubeRadius(Real tubeR) {
+    DecorativeTorusRep::downcast(*rep).setTubeRadius(tubeR); return *this;
+}
+
+/////////////////////
+// DECORATIVE ARROW //
+/////////////////////
+
+DecorativeArrow::DecorativeArrow(const Vec3& startPoint, const Vec3& endPoint, Real tipLength)
```

Comment:
Line too long. Suggested formatting:
```
DecorativeArrow::DecorativeArrow
   (const Vec3& startPoint, const Vec3& endPoint, Real tipLength) {
    rep = new DecorativeArrowRep(startPoint, endPoint, tipLength);
    rep->setMyHandle(*this);
}
```
Other ways are fine too if you think that's ugly.